### DISCUSSION TITLE
User type can't be blank We require applicants to use a school-issued email address to apply.

### DISCUSSION THREAD

**Discussion by saadriz1996** [community/community#120120] [GitHub Education]
### Select Topic Area

Question

### Body

![Screenshot 2024-04-18 215336](https://github.com/community/community/assets/167321820/59e00dfe-caa7-44b8-9acd-cf5f3b917ac2)You have an outstanding discount request for @saadriz1996. We will get back to you soon.

how do i get rid of this error?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @saadriz1996  !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the GitHub Education category. I've gone ahead and moved it for you. Good luck!
